### ISSUE TITLE
[SI-REVIEW-FALSE] 1.4.5 Images of Text - Make sure images do not contain text

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#927]
## Problem Description per Siteimprove

"This warning will always appear for images on your pages because you need to manually make sure that there is no text in images."

## Page Description

- Page Wrapper with Print Logo and Be Boulder logo
- Any basic or otherwise page that has an image inserted into the body field. 
  - The Content List page
  - The Photo Gallery
  - The Article
  - The Article List Page
- Any block that has an image tag inserted. 
  - The hero unit block
  - The content grid block
  - The slider block
  - etc

## What we're doing

- Any < img > tag will generate this error.
- The print logo < img src="{path}/print-logo.png" alt="University of Colorado Boulder" >
- The BeBoulder logo: < img src="{path}/be-boulder-white.png" alt="University of Colorado Boulder" class="beboulder" style="z-index: 2147483646; position: relative;" >

## Solution

This is not something we can fix.  Could be mitigated with https://github.com/CuBoulder/express/issues/926

## Images that did not throw this error but should have

**Video Reveal** block does not generate this error because that image is a background image. It has no alt tag or any way for a screen reader to know there's an image there.
